protos/dcs/mission/v0/mission.proto

```diff
@@ -471,3 +496,29 @@ message GetScenarioCurrentTimeRequest {
 message GetScenarioCurrentTimeResponse {
   string datetime = 1;
 }
+
+message AddMissionCommandRequest {
+  string name = 1;
+  repeated string path = 2;
```

Comment:
How'd you feel about removing `AddMissionCommandSubMenuRequest` in favour of allowing an empty path for root elements?